### ISSUE TITLE
Allow to share dev home configs

### ISSUE THREAD

**Issue by chrisribe** [microsoft/devhome#804]
### Suggested new feature or improvement

I hope we can share dev home configs with other team members.

### Scenario

This would be great for team member onboarding. Some configs can be tedious and error prone

### Additional details

Bonus points configs now document required configs for different teams

**Comment by denelon** [comment]
The WinGet configuration files are just YAML files. What kind of a sharing experience are you thinking of?

**Comment by chrisribe** [comment]
You are right those are YAML files so sending them or having them in a repo would do.

But if you could let's say create a series of configs ex: front-end-cfg, back-end-cfg, intern-cfg, qa-cfg 
It would be nice to simply login with your Github account create/change configs, group them and share that directly.

It could even simply be that this creates a private devhome-save repo with your settings and files.
So any change you make get's pushed to your repo. When sharing you could simply reference the repo url ask for permissions or access (if private).

**Comment by chrisribe** [comment]
> @chrisribe you're probably going to want to add your 👍 to:
> 
> * [Create a config file based on an existing machine #876](https://github.com/microsoft/devhome/issues/876)

Yes this is what users expect, something to help a dev clone an existing setup via the UI.
DevHome assumes users are experts of WinGet. DevHome should make things easy to do.

**Comment by chrisribe** [comment]
Should I close this issue as there is already some work that needs to be done before this is even possible.
This issue is more a "I was expecting more" / "I have a dream".

**Comment by denelon** [comment]
I'm sure others will have thoughts. I don't see a reason to close this issue if we get more 👍 then it's likely something we should figure out via the Dev Home UX.
